scripts/azure-dev/AzureDev.Workstation.psm1

```
@@ -0,0 +1,1611 @@
+
+    [switch]$Optional
+  )
+
+  if ($Context.Yes) {
+    return -not $Optional
+  }
+  $answer = Read-Host "$Prompt [y/N]"
+  return $answer -match '^(?i:y|yes)$'
+}
+
+function Resolve-AzureDevWorkstationName {
```

PSScriptAnalyzer (1.25.0)

[info] The cmdlet 'Resolve-AzureDevWorkstationName' does not have a help comment.

(PSProvideCommentHelp)